src/main/java/io/confluent/kafkarest/resources/v2/PartitionsResource.java

```diff
@@ -0,0 +1,179 @@
+/**
```

Comment:
Is this basically just duplicating the v1 PartitionsResource? I didn't think we needed anything to change from there -- shouldn't we just put v2 on the existing resource?